project/Build.scala

```diff
@@ -857,12 +857,12 @@ object DottyInjectedPlugin extends AutoPlugin {
   }
 
   private def projectChecks(): Unit = {
-    val scalaScala = new File("scala-scala")
-    if (!scalaScala.exists()) {
+    val submodules = List(new File("scala-backend"), new File("scala-scala"), new File("collection-strawman"))
+    if (!submodules.forall(_.exists)) {
       println(
```

Comment:
You shouldn't use println in the sbt build, see http://www.scala-sbt.org/0.13/docs/Howto-Logging.html#Log+messages+in+a+task for the proper way to do it. This should probably be logged at the error level.